lists2safebrowsing.py

```diff
@@ -454,37 +457,75 @@ def get_tracker_lists(config, section, chunknum):
     return output_file, log_file
 
 
+def edit_config(config, section, option, old_value, new_value):
+    current = config.get(section, option)
+    edited_config = current.replace(old_value, new_value)
+    config.set(section, option, edited_config)
+    print('Edited {opt} in {sect} to: {new}'.format(
+        opt=option, sect=section, new=config.get(section, option))
+    )
+
+
+def version_configurations(config, section, version, revert=None):
+    initial_disconnect_url_val = 'master'
+    initial_s3_key_value = 'tracking/'
+    if revert:
+        if config.has_option(section, 'disconnect_url'):
+            edit_config(
+                config, section, option='disconnect_url',
+                old_value=version, new_value=initial_disconnect_url_val)
+        if config.has_option(section, 's3_key'):
+            versioned_key = 'tracking/{ver}/'.format(ver=version)
+            edit_config(
+                config, section, option='s3_key',
+                old_value=versioned_key, new_value=initial_s3_key_value)
```

Comment:
nit: Can this move into a `def revert_configuration(config, section, version, initial_values)` ?